test/switchlevels.c

```diff
@@ -107,10 +107,11 @@ static int compress_chunk(PREFIX3(stream) *strm, int level, int size, int last)
     return ret;
 }
 
-/* ===========================================================================
- * Usage:  switchlevels [-w bits] level1 size1 [level2 size2 ...]
- *   -w : window bits
- */
+void show_help(void)
+{
+    printf("Usage: switchlevels [-w bits] level1 size1 [level2 size2 ...]\n\n" \
+           "  -w : window bits\n\n");
```

Comment:
This should mention valid range for window bits...

Reply:
Fixed.